deployments/kubehound/graph/Dockerfile

```diff
@@ -59,9 +59,22 @@ ENV gremlinserver.metrics.slf4jReporter.enabled=false
 ENV gremlinserver.metrics.graphiteReporter.enabled=false
 ENV gremlinserver.metrics.csvReporter.enabled=false
 
+# Add safety net settings to prevent OOM and other issues
+ENV janusgraph.query.force-index=true
+ENV janusgraph.cluster.max-partitions=512
+ENV janusgraph.query.batch=true
+ENV janusgraph.query.hard-max-limit=10000
+ENV janusgraph.query.smart-limit=true
+
+# Enable cache
+ENV janusgraph.cache.db-cache=true
+ENV janusgraph.cache.db-cache-clean-wait=20
+ENV janusgraph.cache.db-cache-time=180000
```

Comment:
From what I can see in the [doc](https://docs.janusgraph.org/operations/cache/), this is in ms, which means we cache data for 3min in the "database level cache".
Since we ingest on a low frequency (orders of hours/days), shouldn't we enable that for much longer (like, hours at least?)

Reply:
From what I understand from the janusgraph doc, this specific cache doesn't store (only?) the results (since this would be in the transaction cache?) of the query but also the vertices/nodes for frequent access ?
> In contrast to the transaction level caches, the database level caches do not expire immediately after closing a transaction. Hence, the database level cache significantly speeds up graph traversals for read heavy workloads across transactions. A read looks up in transaction-level cache first, and then database-level cache.

> If there is only one JanusGraph instance accessing the storage backend or if this instance is the only one modifying the graph, the cache expiration can be set to 0 which disables cache expiration. This allows the cache to hold elements indefinitely (unless they are evicted due to space constraints or on update) which provides the best cache performance. Since no other JanusGraph instance is modifying the graph, there is no danger of holding on to stale data.

Which to me, indicated that, with the `inmemory` backend, we should likely use no expiration as much as we can?

Reply:
I removed the cache instructions.